src/pybind/mgr/dashboard/module.py

```diff
@@ -362,6 +375,9 @@ def handle_command(self, inbuf, cmd):
         if cmd['prefix'] == 'dashboard create-self-signed-cert':
             self.create_self_signed_cert()
             return 0, 'Self-signed certificate created', ''
+        if cmd['prefix'] == 'dashboard update-grafana-dashboards':
```

Comment:
Why not `dashboard grafana dashboards update`? Nothing really serious, but to keep some hierarchical order: if we add a new command to grafana dashboards, it can be simply nested (e.g.: `dashboard grafana dashboards **remove**`?).